packages/config/config/devices/0x0330/zv2835rac-nf.json

```diff
@@ -0,0 +1,271 @@
+{
+	"manufacturer": "Sunricher",
+	"manufacturerId": "0x0330",
+	"label": "ZV2835RAC-NF",
+	"description": "Knob Smart Dimmer",
+	"devices": [
+		{
+			"productType": "0x0200",
+			"productId": "0xd017"
+		}
+	],
+	"firmwareVersion": {
+		"min": "0.0",
+		"max": "255.255"
+	},
+	"supportsZWavePlus": true,
+	"paramInformation": {
+		"2": {
+			"label": "State After Power Failure",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 2,
+			"defaultValue": 2,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Off",
+					"value": 0
+				},
+				{
+					"label": "On",
+					"value": 1
+				},
+				{
+					"label": "Previous State",
+					"value": 2
+				}
+			]
+		},
+		"3": {
+			"label": "Send Basic Report",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 1,
+			"defaultValue": 1,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Disable",
+					"value": 0
+				},
+				{
+					"label": "Enable",
+					"value": 1
+				}
+			]
+		},
+		"4": {
+			"label": "Default fade time",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 255,
+			"defaultValue": 0,
+			"unsigned": true,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Instantly",
+					"value": 0
+				},
+				{
+					"label": "from 1 to 127 seconds",
+					"value": 1
+				},
+				{
+					"label": "1 to 127 minutes",
+					"value": 80
+				},
+				{
+					"label": "Factory default duration. (3 second)",
+					"value": 255
+				}
+			]
```

Comment:
I guess you're supposed to enter the values here, so this definition doesn't work. You can leave the Instantly and Factory default options. The rest with changing unit can be done as a description:
```
Values 1-127 = seconds; 128-254 = minutes (minus 127)
```
but you'll need `"allowManualEntry": true,`